network/p2pNetwork_test.go

```diff
@@ -794,8 +804,14 @@ func TestP2PHTTPHandler(t *testing.T) {
 func TestP2PRelay(t *testing.T) {
 	partitiontest.PartitionTest(t)
 
+	if strings.ToUpper(os.Getenv("CIRCLECI")) == "TRUE" {
```

Comment:
Are these regression test cases covered elsewhere? Disabling on CI could imply we no longer have coverage here.